TestColorPair.cpp

```diff
@@ -0,0 +1,52 @@
+#include <assert.h>
+#include <iomanip>
+#include <iostream>
+#include<sstream>
+#include <vector>
+#include "ColorPair.h"
+
+std::string trim(std::string m) {
```

Comment:
Normally trim means removing whitespace at the beginning and end.
But this function removes all whitespace, is that right?
Also, when the test starts processing strings, it's time to think if the code can be simplified. E.g., by using an off-the-shelf table-formatter?

Reply:
will try